LICENSE-3rdparty.csv

```diff
@@ -0,0 +1,6 @@
+Component,Origin,License,Copyright
+gemfile, rubocop, MIT, Copyright (c) 2012-19 Bozhidar Batsov
+gemfile, rake, MIT, Copyright (c) Jim Weirich
+gemfile, rspec, MIT, Copyright © 2009 Chad Humphries, David Chelimsky Copyright © 2006 David Chelimsky, The RSpec Development Team Copyright © 2005 Steven Baker
```

Comment:
The extra commas in the last field will result in a GitHub message when viewing the file. Other than that LGTM.